velox/functions/prestosql/tests/TimestampWithTimeZoneCastTest.cpp

```diff
@@ -21,16 +21,20 @@ using namespace facebook::velox;
 
 class TimestampWithTimeZoneCastTest : public functions::test::CastBaseTest {
  public:
-  RowVectorPtr makeTimestampWithTimeZoneVector(
+  VectorPtr makeTimestampWithTimeZoneVector(
       const VectorPtr& timestamps,
       const VectorPtr& timezones) {
```

Comment:
Ditto. It would be easier to receive two functors.

Reply:
Refactored.